applications/logan_progress_report/cover_letter.tex

```diff
@@ -0,0 +1,39 @@
+\documentclass{letter}
+\usepackage{hyperref}
+\signature{Zach Latta \\ Executive Director}
+\address{
+  hackEDU \\
+  44 Tehama Street \\
+  San Francisco, CA, 94105
+}
+\begin{document}
+
+\begin{letter}{
+    Richard Logan \\
+    Director \\
+    The Reva \& David Logan Foundation \\
+    980 N. Michigan Ave. \#1122 \\
+    Chicago, IL, 60611
+  }
+  \opening{Dear Mr. Richard Logan,}
+
+  I'm pleased to submit our mid-year report to the board of The Reva \& David
+  Logan Foundation. The attached document reviews our progress and outlines our
+  plans for the coming six months, with an eye on 2016 and beyond.
+
+  In brief: we created an early prototype of the ``club in a box'', grew from 36
+  to 47 schools, and doubled our presence in foreign countries. Our report
+  provides details, reflecting on our initial proposal and goals.
+
+  This upcoming semester we are focusing our attention on 12 clubs---an
+  initiative we are calling ``Labs''---to refine a working version of our ``club
```

Comment:
Refine implies that you already have a working playbook, at least to me. I don't think we do, but either way it is just technicalities... Because there are "hackEDU Clubs" running already without it!